apps/cookbook/src/app/page/side-nav/side-nav.component.ts

```diff
@@ -14,14 +15,12 @@ import { filter } from 'rxjs/operators';
 import { kebabToTitleCase } from '@kirbydesign/designsystem';
 
 import { routes } from '../../showcase/showcase.routes';
```

Comment:
Could we import app routes for resources same way as showcase routes?
```suggestion
import { appRoutes } from '../../app.routes';
import { showCaseRoutes } from '../../showcase/showcase.routes';
```